kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -103,27 +105,68 @@ window.SidebarView = BaseView.extend({
 
         this.$('.sidebar-content').append(this.topic_node_view.el);
 
+        this.set_sidebar_back();
+
         return this;
     },
 
-    resize_sidebar: _.debounce(function() {
+    resize_sidebar: function() {
+        if ($(window).width() < 1260)
+            this.resize_for_narrow();
+
+        else
+            this.resize_for_wide();
+    },
+
+    resize_for_narrow: _.debounce(function() {
         var current_level = this.state_model.get("current_level");
         // TODO(jamalex): have this calculated dynamically
         var column_width = 200; // this.$(".topic-container-inner").width();
         var last_column_width = 400;
```

Comment:
Right, so even the one column can still overflow really small screens. Perhaps we can set the `.topic-container-inner` width property using css media rules and, below a certain viewport threshold, simply have the column take up the full width of the screen?